### PR TITLE
Feature/navigation initial

### LINKED ISSUE #1: Start learning React Native
- Start some sort of online course/tutorial on React Native

### LINKED ISSUE #3: Create initial FE code base
- Spin up React Native App and start to have discussions on how Components will be broken out

### LINKED ISSUE #4: Clickable house
- A user should be able to click on the house and be routed to a birds eye view of the house

### PR DESCRIPTION
**Pull request by brycemara**
## Description

- Navigation in Native
- Passing of props to components

## Motivation and Context

- Now the user can move between views!

## How Has This Been Tested?

- No testing done yet

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by Estaffieri — APPROVED** [review]
So glad we got this navigation going, and figured out passing props!

File: src/App/App.js [hunk 1]
```diff
@@ -1,48 +1,27 @@
...
+      <NavigationContainer>
+        <Stack.Navigator>
+          <Stack.Screen name="HomePage" component={HomePage} />
+          <Stack.Screen name="Overview" component={Overview} />
+          <Stack.Screen name="Room" component={Room} />
+          <Stack.Screen name="Memory" component={Memory} />
+        </Stack.Navigator>
+      </NavigationContainer>
+    );
+  }
```
Comment (Codo-Baggins) [inline src/App/App.js:24]: Very nice here.

File: src/HomePage/HomePage.js [hunk 2]
```diff
@@ -1,29 +1,39 @@
...
+    super(props);
+    this.state = {
+      rooms: [],
+    };
+  }
+  resolvedRooms = async () => {
+    await getAllRooms(1)
+      .then((data) => this.setState({ rooms: data.data }))
+      .then((response) =>
+        this.props.navigation.navigate("Overview", { rooms: this.state.rooms })
```
Comment (Codo-Baggins) [inline src/HomePage/HomePage.js:16]: Is the second argument for the "navigate" method the props that are being passed to Overview?

File: src/HomePage/HomePage.js [hunk 3]
```diff
@@ -1,29 +1,39 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      rooms: [],
+    };
+  }
+  resolvedRooms = async () => {
+    await getAllRooms(1)
+      .then((data) => this.setState({ rooms: data.data }))
+      .then((response) =>
```
Comment (cameronRomo) [inline src/HomePage/HomePage.js:15]: Interesting way to pass props down. So glad we got this figured out.

File: src/Stack/Stack.js [hunk 4]
```diff
@@ -1,22 +1,19 @@
...
+//   return (
+//     <NavigationContainer>
+//       <Stack.Navigator>
+//         {/* <Stack.Screen name='App' component={App} /> */}
+//         <Stack.Screen name="HomePage" component={HomePage} />
+//         <Stack.Screen name="Overview" component={Overview} />
+//       </Stack.Navigator>
+//     </NavigationContainer>
+//   );
+// };
```
Comment (Codo-Baggins) [inline src/Stack/Stack.js]: We should delete this file completely.

**Review by Codo-Baggins — APPROVED** [review]
Nice job getting this working!  Now we need to just fill the app with the proper data!

**Review by cameronRomo — APPROVED** [review]
I'm so glad we got our issues figured out here! Lots of growth in regards to the navigation and passing props around 🌱.

**Pull request merged**